package.yaml

```diff
@@ -30,6 +30,9 @@ when:
   - condition: impl(ghc < 8.6)
     ghc-options:
       - -Wno-redundant-constraints
+  - condition: impl(ghc >= 9.8)
+    ghc-options:
+      - -Wno-missing-role-annotations
```

Comment:
Sorry to be a stickler, but the list of conditionals are well ordered new-to-old, so can you put this first instead of last?